### ISSUE TITLE
UI is not rendering in Chrome/Mozilla/Opera browsers but working in IE 11

### ISSUE THREAD

**Issue by shaikficus** [oracle/apps-cloud-ui-kit#2]
Using Jdeveloper 12.1.3, Running RDK 13 by running stand alone or deploying to Integrated Weblogic Server.
Issue in Screen Rendering
Screen shot from IE 11
![image](https://user-images.githubusercontent.com/16458706/31337171-af566e74-ad17-11e7-9686-b04f09bcbc3d.png)
Screen Shot from Opera Browser
![image](https://user-images.githubusercontent.com/16458706/31337243-0c7ab344-ad18-11e7-987c-ba781d1ff875.png)

**Comment by lancysilveira** [comment]
shaikficus, I just tested a freshly pulled copy of the RDK on JDeveloper 12.1.3.0.0 and found this to be working fine on Chrome (62.0.3202.94(64-bit)), Firefox (57.0(64-bit)) and Safari (11.0.1(13604.3.5)). However the error you are experiencing may be related to an issue reported early last month for which a fix was applied on 05-Oct-17. 

So can you do the following things please:

1. Pull the latest RDK code from GitHub (if your code if from before 06-Oct-17)

2. Then using your JDeveloper 12c1300, apply the workaround specific to this version of JDeveloper as mentioned in the README file of the ADF RDK. 

3. Now generate the ADF Libraries in order for - DemoData, UIKitCommon, DemoCRM, DemoFIN, DemoHCM.

4. Then generate the DemoMaster WAR.

5. Right-click Welcome.jspx and choose Run.

**Comment by lancysilveira** [comment]
I suspect the issue you experienced is related to #1. Closing for now as I do not expect a fix being required. Please re-open if issue continues after trying the above.

**Comment by shaikficus** [comment]
Thank you Lancysilveira,

Working like charm after pulling the latest repo and tested in 12.1.3 Jdeveloper with changes as suggested in readme.

Due to my own interest I opened the application in 12.2.1.2 Jdeveloper and added JSF libraries to 2.2, Everthing is working fine except when I open any node in the home page, getting the below error java.el expression not found '#{comp.pageStickyTitle}'on PageHeaderDC Decalrative component.
I verified the bean and added below variable to it  'private String PageStickyTitle;'.

I'm not sure how it worked in 12.1.3 and didn't 12.2.1.2.

Again, Thanks for your update.
